lib/dialects/abstract/query-generator.js

```diff
@@ -403,7 +403,8 @@ const QueryGenerator = {
 
     for (const key in attrValueHash) {
       const value = attrValueHash[key];
-      values.push(this.quoteIdentifier(key) + '=' + this.quoteIdentifier(key) + operator + this.escape(value));
+      const negativeValueIsNotSqlComment = (typeof value === 'number' && value < 0) ? ' ' : '';
+      values.push(this.quoteIdentifier(key) + '=' + this.quoteIdentifier(key) + operator + negativeValueIsNotSqlComment + this.escape(value));
```

Comment:
`negativeValueIsNotSqlComment` is tooooo verbose., just add a space after operator permanently